<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RootGUI</class>
 <widget class="QWidget" name="RootGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="raytracer.qrc">
    <normaloff>:/raytracer/Resources/Icon.jpg</normaloff>:/raytracer/Resources/Icon.jpg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QTabWidget" name="mTabGrp">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="Loader">
        <attribute name="title">
         <string>File Loader</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLineEdit" name="mSceneFileLoc">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="mSceneFileButton">
                 <property name="minimumSize">
                  <size>
                   <width>151</width>
                   <height>23</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>151</width>
                   <height>23</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Select scene properties XML</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLineEdit" name="mObjFileLoc">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="mObjFileButton">
                 <property name="minimumSize">
                  <size>
                   <width>151</width>
                   <height>23</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>151</width>
                   <height>23</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Select objects XML</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="mEffectLabel">
                 <property name="minimumSize">
                  <size>
                   <width>151</width>
                   <height>16</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>151</width>
                   <height>16</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Select Rendering Effect</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="mEffectComboBox">
                 <property name="minimumSize">
                  <size>
                   <width>151</width>
                   <height>22</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>151</width>
                   <height>22</height>
                  </size>
                 </property>
                 <item>
                  <property name="text">
                   <string>Simple Ray Trace</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Depth of field</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Motion Blur effect</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Ambient occlusion</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Animation</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>All Effects</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="mEvnMapLabel">
                 <property name="minimumSize">
                  <size>
                   <width>151</width>
                   <height>16</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>151</width>
                   <height>16</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Environment map availability</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="mEnvMapComboBox">
                 <property name="minimumSize">
                  <size>
                   <width>151</width>
                   <height>22</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>151</width>
                   <height>22</height>
                  </size>
                 </property>
                 <item>
                  <property name="text">
                   <string>disable</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>enable</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>308</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1">
            <spacer name="verticalSpacer_7">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>308</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1">
            <spacer name="horizontalSpacer_10">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>308</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="0">
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="1">
            <spacer name="verticalSpacer_6">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="mLoadInputsButton">
               <property name="minimumSize">
                <size>
                 <width>75</width>
                 <height>23</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>75</width>
                 <height>23</height>
                </size>
               </property>
               <property name="text">
                <string>Load Inputs</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="mRenderButton">
               <property name="minimumSize">
                <size>
                 <width>75</width>
                 <height>23</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>75</width>
                 <height>23</height>
                </size>
               </property>
               <property name="text">
                <string>Render</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>128</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="Canvas" name="Viewer">
        <attribute name="title">
         <string>Render Viewer</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="SceneDesc">
        <attribute name="title">
         <string>Scene Description</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTreeWidget" name="mSceneTree">
           <property name="columnCount">
            <number>3</number>
           </property>
           <column>
            <property name="text">
             <string notr="true">Property Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string notr="true">Attributes</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string notr="true">Value</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="ObjectList">
        <attribute name="title">
         <string>Objects</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QTreeWidget" name="mObjTree">
           <property name="columnCount">
            <number>3</number>
           </property>
           <column>
            <property name="text">
             <string notr="true">Property Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Attributes</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Value</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="mProgressBar">
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="textDirection">
        <enum>QProgressBar::BottomToTop</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Canvas</class>
   <extends>QWidget</extends>
   <header>canvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="raytracer.qrc"/>
 </resources>
 <connections/>
</ui>
